frontend/src/container/MetricsApplication/index.tsx

```diff
@@ -1,54 +1,108 @@
-import { Tabs } from 'antd';
+import RouteTab from 'components/RouteTab';
+import ROUTES from 'constants/routes';
 import React from 'react';
+import { generatePath, useParams } from 'react-router-dom';
+import { useLocation } from 'react-use';
 import { Widgets } from 'types/api/dashboard/getAll';
 
 import ResourceAttributesFilter from './ResourceAttributesFilter';
-import Application from './Tabs/Application';
 import DBCall from './Tabs/DBCall';
 import External from './Tabs/External';
+import Overview from './Tabs/Overview';
 
-const { TabPane } = Tabs;
+const getWidget = (query: Widgets['query']): Widgets => {
+	return {
+		description: '',
+		id: '',
+		isStacked: false,
+		nullZeroValues: '',
+		opacity: '0',
+		panelTypes: 'TIME_SERIES',
+		query,
+		queryData: {
+			data: [],
+			error: false,
+			errorMessage: '',
+			loading: false,
+		},
+		timePreferance: 'GLOBAL_TIME',
+		title: '',
+		stepSize: 60,
+	};
+};
+
+function OverViewTab(): JSX.Element {
+	return <Overview getWidget={getWidget} />;
+}
+
+function DbCallTab(): JSX.Element {
+	return <DBCall getWidget={getWidget} />;
+}
+
+function ExternalTab(): JSX.Element {
+	return <External getWidget={getWidget} />;
+}
 
 function ServiceMetrics(): JSX.Element {
-	const getWidget = (query: Widgets['query']): Widgets => {
-		return {
-			description: '',
-			id: '',
-			isStacked: false,
-			nullZeroValues: '',
-			opacity: '0',
-			panelTypes: 'TIME_SERIES',
-			query,
-			queryData: {
-				data: [],
-				error: false,
-				errorMessage: '',
-				loading: false,
-			},
-			timePreferance: 'GLOBAL_TIME',
-			title: '',
-			stepSize: 60,
-		};
+	const { search } = useLocation();
+	const { servicename } = useParams<{ servicename: string }>();
+
+	const searchParams = new URLSearchParams(search);
+	const tab = searchParams.get('tab');
+
+	const overMetrics = 'Overview Metrics';
+	const dbCallMetrics = 'Database Calls';
+	const externalMetrics = 'External Calls';
```

Comment:
Let's use the values from here https://github.com/SigNoz/signoz/blob/f084637f843ae055313e8995f24461148ac45b55/frontend/public/locales/en-GB/routes.json#L6

Reply:
use Translation is re-rendering twice here in which api call are making twice

Reply:
Oh I see

Reply:
https://github.com/isaachinman/next-i18next/issues/654